private_notices/serializers.py

```diff
@@ -9,10 +9,11 @@ class NotificationSerializer(serializers.Serializer):
 
 
 class PrivateNoticeViewSerializer(serializers.ModelSerializer):
-    notification = NotificationSerializer(many=True)
-    sender = serializers.SerializerMethodField('return_sender')
+    pnotice = serializers.CharField(max_length=500)
```

Comment:
lets rename the variable `pnotice` with `private_notice` to make it more verbose.